bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -340,6 +346,52 @@ impl Room {
         });
     }
 
+    pub fn create_poll(
+        &self,
+        question: String,
+        answers: Vec<String>,
+        max_selections: u64,
```

Comment:
```suggestion
        max_selections: u8,
```

The maximum number of answers is capped at 20 anyways, right? Could also use `u16` or `u32`, but in any case it should be something that's smaller than `UInt` IMHO.

Reply:
Yep `20`.
u8 fells better. 👍🏻
[Applied](https://github.com/matrix-org/matrix-rust-sdk/pull/2391/commits/96e4e22be012e8dff804be23ccb1d4a97ab44e58)